guides/migrations/2.0.x_2.1.0.md

```diff
@@ -0,0 +1,125 @@
+```elixir
+config :maru, :test, false
+```
+
+## Environmental configs (`config/{prod|dev|test}.exs`)
+
+### `Phoenix` in place of `Maru`
+As mentioned earlier, MongoosePush uses `Phoenix` and the config has to be updated to reflect that change.
+Old `Maru` config entry:
+```elixir
+config :maru, MongoosePush.Router,
+  versioning: [
+    using: :path
+  ],
+  https: [
+    bind_addr: {:system, :string, "PUSH_HTTPS_BIND_ADDR", "127.0.0.1"},
+    port: {:system, :integer, "PUSH_HTTPS_PORT", 8443},
+    keyfile: {:system, :string, "PUSH_HTTPS_KEYFILE", "priv/ssl/fake_key.pem"},
+    certfile: {:system, :string, "PUSH_HTTPS_CERTFILE", "priv/ssl/fake_cert.pem"},
+    cacertfile: {:system, :string, "PUSH_HTTPS_CERTFILE", "priv/ssl/fake_cert.pem"},
+    acceptors: {:system, :integer, "PUSH_HTTPS_ACCEPTORS", 100},
+    otp_app: :mongoose_push
+  ]
+```
+It should be removed completely and in its place there must be the new, `Phoenix` config entry:
+```elixir
+config :mongoose_push, MongoosePushWeb.Endpoint,
+  https: [
+    ip: {
+      :system,
+      # Custom type parser (Phoenix needs erlang-inet-style IP address)
+      {MongoosePush.Config.Utils, :parse_bind_addr, []},
+      "PUSH_HTTPS_BIND_ADDR",
+      {127, 0, 0, 1}
+    },
+    port: {:system, :integer, "PUSH_HTTPS_PORT", 8443},
+    keyfile: {:system, :string, "PUSH_HTTPS_KEYFILE", "priv/ssl/fake_key.pem"},
+    certfile: {:system, :string, "PUSH_HTTPS_CERTFILE", "priv/ssl/fake_cert.pem"},
+    cacertfile: {:system, :string, "PUSH_HTTPS_CERTFILE", "priv/ssl/fake_cert.pem"},
+    protocol_options: [
+      # https://ninenines.eu/docs/en/cowboy/2.5/manual/cowboy_http/
+    ],
+    transport_options: [
+      # https://ninenines.eu/docs/en/ranch/1.6/manual/ranch_tcp/
+      # https://ninenines.eu/docs/en/ranch/1.6/manual/ranch_ssl/
+      num_acceptors: {:system, :integer, "PUSH_HTTPS_ACCEPTORS", 100}
+    ],
+    otp_app: :mongoose_push
+  ],
+  debug_errors: false,
+  code_reloader: false,
+  check_origin: true,
+  server: true
+  ```
+  As you can see, there is no change in the variables used to parse the options.
+
+### Logging
+Old logging entry looked like that:
+```elixir
+config :mongoose_push, loglevel: {:system, :atom, "PUSH_LOGLEVEL", :info}
+```
+
+New logging entry may look like that:
+```elixir
+config :mongoose_push, :logging,
+  level: {:system, :atom, "PUSH_LOGLEVEL", :info},
+  format: {:system, :atom, "PUSH_LOGFORMAT", :json}
+```
+
+The structure changed a little and there is a new variable, `PUSH_LOGFORMAT`. It defaults to `:json` for the `prod` environment, and to `:logfmt` for the `dev` and `test` environments. The `logfmt` backend was changed to support structured logging. Please note that if you had a MongoosePush node configured with a log aggregation backend it probably needs to be reconfigured to match the current format of the log messages. You might also want to consider using the second standard, `JSON`, that MongoosePush now supports.
```

Comment:
I find it more readable and easier to change later if the text in markdown files follow the rule "one sentence per line". Could you split this long line into several ones, each containing one sentence?

Reply:
I've split the text into sentence per line in this paragraph and in the remaining ones.